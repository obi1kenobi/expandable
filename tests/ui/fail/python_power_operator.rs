#[expandable::expr]
macro_rules! square {
    ($e:expr) => { $e ** $e }
}

fn main() {}