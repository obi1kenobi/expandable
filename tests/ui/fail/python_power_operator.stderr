error: Potentially invalid expansion. Expected an identifier.
 --> tests/ui/fail/python_power_operator.rs:5:14
  |
5 |         $e * *$e
  |              ^
